app/controllers/ArticlesController.scala

```diff
@@ -16,11 +16,13 @@ class ArticlesController @Inject() extends Controller {
 
   def getArticles = Action { implicit request =>
     val optAuthor = request.getQueryString("author")
+    val optDate = request.getQueryString("date")
     val optContent = request.getQueryString("content")
     val optTitle = request.getQueryString("title")
 
     val filteredArticles = Articles.getArticles
       .filter( article => optAuthor.fold(true)(author => author == article.author))
+      .filter( article => optDate.fold(true)(date => date == article.date))
```

Comment:
would have to be called with values like `/api/articles?date=2017-03-19T19:31:11.068Z` to work. Which can only return one result really.